test/python3/libdnf/support/support.py

```diff
@@ -0,0 +1,47 @@
+import os
+import shutil
+import tempfile
+import unittest
+
+import libdnf
+
+
+PROJECT_BINARY_DIR = os.environ["PROJECT_BINARY_DIR"]
+PROJECT_SOURCE_DIR = os.environ["PROJECT_SOURCE_DIR"]
+
+
+class LibdnfTestCase(unittest.TestCase):
+    def setUp(self):
+        self.base = libdnf.base.Base()
+
+        self.cachedir = tempfile.mkdtemp(prefix="libdnf-test-python3-")
+        self.base.get_config().cachedir().set(libdnf.conf.Option.Priority_RUNTIME, self.cachedir)
+
+        self.repo_sack = self.base.get_repo_sack()
+        self.sack = self.base.get_rpm_package_sack()
+
+    def tearDown(self):
+        shutil.rmtree(self.cachedir)
+
+    def _add_repo(self, repoid, repo_path):
+        repo = self.repo_sack.new_repo(repoid)
+        repo.get_config().baseurl().set(libdnf.conf.Option.Priority_RUNTIME, "file://" + repo_path)
+        repo.load()
+        self.sack.load_repo(repo.get(),
+            libdnf.rpm.PackageSack.LoadRepoFlags_USE_FILELISTS |
+            libdnf.rpm.PackageSack.LoadRepoFlags_USE_OTHER |
+            libdnf.rpm.PackageSack.LoadRepoFlags_USE_PRESTO |
+            libdnf.rpm.PackageSack.LoadRepoFlags_USE_UPDATEINFO
+        )
+
+    def add_repo_rpm(self, repoid):
```

Comment:
The new code completely removes all comments what it does. Please can you add descriptions what each method does and also keep some comments in the code?

Reply:
Done